canonical-websites/snapcraft.io
# Snap list columns

# Done

Fixes https://github.com/canonical-websites/snapcraft.io/issues/759

- Added percentage widths to the `th` elements

# QA

- Pull the branch
- `./run`
- Visit http://0.0.0.0:8004/account/snaps
- Make sure the table columns are sensible widths
